jbpm-db-scripts/src/test/filtered-resources/META-INF/persistence.xml

```diff
@@ -4,6 +4,29 @@
              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
              xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/persistence http://xmlns.jcp.org/xml/ns/persistence/persistence_2_1.xsd">
 
+  <persistence-unit name="testTypes" transaction-type="JTA">
+    <provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>
+    <jta-data-source>jdbc/testDS1</jta-data-source>
+    <class>org.jbpm.persistence.types.TestEntity</class>
+    <exclude-unlisted-classes>true</exclude-unlisted-classes>
+    <properties>
+      <property name="hibernate.max_fetch_depth" value="3"/>
+      <!-- No value for hbm2ddl.auto, because tests manage DDL themselves. -->
+      <property name="hibernate.hbm2ddl.auto" value=""/>
```

Comment:
Sorry @elguardian - this needs to be changed as well, since we are not providing any DDL script for this entity creation, thus `hbm2ddl.auto` has to be set to `create-drop` or `update`

```suggestion
      <property name="hibernate.hbm2ddl.auto" value="create-drop"/>
```

Reply:
done